### ISSUE TITLE
Outside contributor would not know how to start

### ISSUE THREAD

**Issue by yosefm** [OpenPTV/openptv#184]
README files are outdatd, at least regarding tests and build process. Trying to run "make verify" as the README instructs can take up an hour before you find (by asking an active developer)  that the tests are now all done through py_bind. If one has to ask, you already lost 90% of potential contributors. They wouldn't ask, they'll just go away. Just like with the installation.

**Comment by alexlib** [comment]
@yosefm - I find this page in a reasonable condition 
https://github.com/OpenPTV/openptv

and it points to the documentation on readthedocs. 

do you think it is necessary to keep also https://github.com/OpenPTV/openptv/blob/master/liboptv/README.txt 

that is not in sync with the documentation? 

I think it'll be more reasonable to remove all README files or point from each one of them to the same central documentation? 

I think this question relates to the main one: do we keep `liboptv` compilation and readiness for C interface or we are fine with C code being just a Python binded and tested? 

What do you think? 

I'd also appreciate @zmbq opininon on this.

Thanks

**Comment by yosefm** [comment]
Yes, I think keeping the liboptv tests and the docs for how to run them is important. When I do dev. work on that level, I should have a tool to directly test the code without all the Python layers. Besides, keeping this library general enough so that someone can use it from other languages is good software engineering. It "keeps you honest" and prevents the UI and the core logic from mixing.
